docs/specs/tripgo.swagger.yaml

```diff
@@ -1757,6 +1757,9 @@ definitions:
         type: integer
       source:
         $ref: '#/definitions/DataSourceAttribution'
+      deepLink:
+        description: deep link to this pod
```

Comment:
Can we have an example for this? It's not clear to me whether this would be a deep-link to a website (which apps might intercept using the "Universal Links" mechanism) or whether this is a deep-link into the provider's app, and the apps might need to check first if they can open this link. If it's the latter, we might also need both `downloadLinkiOS` and `downloadLInkAndroid` for the case that the app is not installed. See also [redmine #10700](
https://redmine.buzzhives.com/issues/10700)

Reply:
we are using this for two providers currently:

- SocialBicycles: sobi://book?hub_id=XXX - opens app on hub XXX booking screen
- GBFS: using the rental_url, which I don't have a sample, @fersagui can you easily get one?

In both cases, don't think we would get app download links.

Reply:
The download links are in the `AppInfo` object. Is that provided as part of `BikePodInfo`?

Reply:
BikePodInfo has a `operatorInfo: CompanyInfo` which in turn has `appInfo: AppInfo`, which then has the download links. So all good.